docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md

```diff
@@ -64,7 +64,7 @@ following limitations currently exist for SOA Suite domains:
 * Only configured clusters are supported.  Dynamic clusters are not supported for
   SOA Suite domains.  Note that you can still use all of the scaling features,
   you just need to define the maximum size of your cluster at domain creation time.
-* Deploying and running SOA Suite domains is supported only in operator versions 2.4.0 and later.
+* Deploying and running SOA Suite domains is supported only in operator version 2.4.0.
```

Comment:
@rosemarymarano, should this be "supported only with" or "supported only on"?  It sounds odd to say "only in" with just one version.

Reply:
@rjeberhard Yep, "supported only with" is correct.

Reply:
@rdas0405 @sbattagi can you help me understand why you are saying only with 2.4.0, rather than 2.4.0 or later please

Reply:
I've been thinking about this more...  We need to get on the same page.  Since this is an open-source project with a short release cycle, the intention was to always move the releases forward -- that is, customers would receive bug fixes by upgrading to newer versions of the operator.  We may need to designate certain versions as having long term support, but that would only be practical for the last release in a given major version line (so, not 2.4.0).  If a SOA customer finds a bug while using 2.4.0, my expectation is that this customer would use 2.5.0.

Reply:
Hi @markxnelson 

This is for SOA early access customers we are suggesting to use operator 2.4.0 only, as with 2.5.0 we have a schema validation failure, which needs a code fix.

Thanks
Samba